### PR TITLE
NCS-532 add endpoint to get psc data

### PR DESCRIPTION
**Pull request by markpit**
Adding new endpoint /confirmation-statement/company/{companyNumber}/persons-of-significant-control 
to get the pscs from the oracle-query-api

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/PscService.java [hunk 1]
```diff
@@ -32,4 +41,11 @@ public PscsApi getPscs(String companyNumber) throws ServiceException {
             throw new ServiceException("Error Retrieving Persons of significant control", e);
         }
     }
+
+    public List<PersonOfSignificantControlJson> getPSCsFromOracle(String companyNumber) throws ServiceException {
+        List<PersonOfSignificantControl> pSCs = oracleQueryClient.getPersonsOfSignificantControl(companyNumber);
```
Comment (mattch1) [inline src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/PscService.java:46]: This need to be camel case?

File: src/test/java/uk/gov/companieshouse/confirmationstatementapi/controller/PersonOfSignificantControlJsonControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    private PersonsOfSignificantControlController personsOfSignificantControlController;
+
+    @Test
+    void testGetPersonsOfSignificantControlOKResponse() throws ServiceException {
+        var pSCs = Arrays.asList(new PersonOfSignificantControlJson(), new PersonOfSignificantControlJson());
+        when(pscService.getPSCsFromOracle(COMPANY_NUMBER)).thenReturn(pSCs);
+        var response = personsOfSignificantControlController.getPersonsOfSignificantControl(COMPANY_NUMBER);
+
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+        assertEquals(pSCs, response.getBody());
```
Comment (mattch1) [inline src/test/java/uk/gov/companieshouse/confirmationstatementapi/controller/PersonOfSignificantControlJsonControllerTest.java:38]: See camel case comment above.

File: src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/PscService.java (hunk 1 above)
Comment (markpit) [reply src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/PscService.java:46]: i thought it was PSCs = pSCs. intellij complains of spelling if i use pscs.

Comment (markpit) [reply src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/PscService.java]: updated, i can live with intellij's spelling warning

**Review by AlfredBwomezi — APPROVED** [review]

**Pull request merged**